### PR TITLE
chore: Cleanup repo

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -13,11 +13,12 @@ learn how: http://kcd.im/pull-request
 
 - `gatsby-plugin-pinterest` version:
 - `node` version:
-- `npm` (or `yarn`) version:
+- `npm` version:
```
Comment (robinmetral) [inline .github/ISSUE_TEMPLATE.md:16]: Why not yarn?

Comment (robinmetral) [reply .github/ISSUE_TEMPLATE.md:16]: Or is this simply automated from kcd-scripts?

Comment (MichaelDeBoey) [reply .github/ISSUE_TEMPLATE.md:16]: I made it equal to `kcd-scripts`

**Pull request merged**